substrate/frame/staking/ah-client/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "pallet-staking-ah-client"
+description = "Pallet handling the communication with staking-rc-client. It's role is to glue the staking pallet (on AssetHub chain) and session pallet (on Relay Chain) in a transparent way."
+license = "Apache-2.0"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
+
+[dependencies]
+codec = { workspace = true, features = ["derive"] }
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+scale-info = { workspace = true, features = ["derive"] }
+sp-runtime = { features = ["serde"], workspace = true }
+sp-staking = { features = ["serde"], workspace = true }
+pallet-authorship = { workspace = true }
+pallet-session = { features = [	"historical",], workspace = true }
+pallet-staking = { workspace = true}	# todo: probably not a good idea to depend on the whole staking pallet
```

Comment:
Is it a bad idea to depend on the whole `pallet-staking` just to use `ExposureOf`?

Reply:
As discussed with @Ank4n, `ExposureOf` is removed in [his PR](https://github.com/paritytech/polkadot-sdk/pull/7424)